src/BaGet.Core.Server/Extensions/NugetBehaviorMiddleware.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Net.Http.Headers;
+
+namespace BaGet.Extensions
+{
+    /// <summary>
+    /// Special middleware that translates the non RFC behavior from NuGet.exe into default Bearer RFC behavior before the default JwtBearer Middleware from the AspNetCore Team is called
+    /// This special rewrite of request/response is executed only if the client is the nuget commandline. Other clients (Web-UI) are expected to be implemented RFC conform!
+    /// </summary>
+    public class NugetBehaviorMiddleware
+    {
+        private readonly RequestDelegate NextRequest;
```

Comment:
Please move the `const` to be before instance members, and please rename private instance fields to `_nextRequest` and `_logger`

Reply:
done!